crypto/java/CachingSample.java

```diff
@@ -158,4 +158,6 @@ public char[] getPassphrase(NAESessionInterface session) {
             return passPhrase;
         }
     }
+}       
```

Comment:
These closing brackets will not come. Please check

Reply:
updated